src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml

```diff
@@ -0,0 +1,155 @@
+        ControlSelected="FilterLabel_ControlSelected">
+      </local:FilterLabel>
+
+      <TextBlock
+        Grid.Column="4"
+        VerticalAlignment="Center"
+        Text="{x:Static resx:Resources.Label_Repository}" />
+      <ComboBox
+        Grid.Column="6"
+        Margin="6,0,0,0"
+        x:Name="_sourceRepoList"
+        VerticalAlignment="Center"
+        DisplayMemberPath="PackageSource.Name"
+        MinWidth="130"
+        MinHeight="22"
+        AutomationProperties.Name="{x:Static resx:Resources.Label_Repository}"
+        SelectionChanged="_sourceRepoList_SelectionChanged">
+        <ComboBox.ToolTip>
+          <ToolTip
+                x:Name="_sourceTooltip">
+            <TextBlock
+                  Text="{Binding}" />
+          </ToolTip>
+        </ComboBox.ToolTip>
+      </ComboBox>
+
+      <Button
+        Grid.Column="7"
+        Margin="12,0,0,0"
+        VerticalAlignment="Center"
+        Padding="0"
+        Style="{StaticResource {x:Static ToolBar.ButtonStyleKey}}"
+        Click="_settingsButton_Click"
+        x:Name="_settingsButton">
+        <Rectangle
+          Width="{Binding ElementName=_self,Path=FontSize,Converter={StaticResource Font122PercentSizeConverter}}"
+          Height="{Binding ElementName=_self,Path=FontSize,Converter={StaticResource Font122PercentSizeConverter}}">
+          <Rectangle.Fill>
+            <DrawingBrush>
+              <DrawingBrush.Drawing>
+                <DrawingGroup>
+                  <DrawingGroup.Children>
+                    <GeometryDrawing
+                          Brush="#00FFFFFF"
+                          Geometry="F1M16,16L0,16 0,0 16,0z" />
+                    <GeometryDrawing
+                          Brush="#FFF6F6F6"
+                          Geometry="F1M15.9434,7.0796L15.8734,6.4596 13.6374,5.6646 14.6544,3.5256 14.2684,3.0376C13.8854,2.5536,13.4464,2.1146,12.9644,1.7316L12.4754,1.3456 10.3354,2.3626 9.5414,0.1286 8.9224,0.057599999999999C8.6194,0.0215999999999994 8.3124,-0.000400000000000844 8.0004,-0.000400000000000844 7.6894,-0.000400000000000844 7.3824,0.0215999999999994 7.0804,0.0565999999999995L6.4614,0.126599999999999 5.6664,2.3626 3.5254,1.3456 3.0374,1.7316C2.5554,2.1126,2.1164,2.5516,1.7324,3.0366L1.3454,3.5246 2.3624,5.6646 0.1274,6.4596 0.0564,7.0796C0.0224000000000011,7.3816 0.000400000000000844,7.6876 0.000400000000000844,7.9996 0.000400000000000844,8.3116 0.0224000000000011,8.6186 0.0564,8.9206L0.1274,9.5396 2.3624,10.3346 1.3454,12.4746 1.7324,12.9626C2.1144,13.4466,2.5534,13.8846,3.0374,14.2676L3.5254,14.6556 5.6664,13.6376 6.4614,15.8726 7.0804,15.9436C7.3824,15.9786 7.6894,15.9996 8.0004,15.9996 8.3134,15.9996 8.6204,15.9786 8.9244,15.9426L9.5424,15.8706 10.3364,13.6366 12.4754,14.6556 12.9644,14.2676C13.4484,13.8836,13.8874,13.4446,14.2684,12.9616L14.6544,12.4736 13.6374,10.3346 15.8734,9.5396 15.9434,8.9206C15.9784,8.6186 16.0004,8.3116 16.0004,7.9996 16.0004,7.6876 15.9784,7.3816 15.9434,7.0796" />
```

Comment:
Not sure if the long line rule can be applied to xaml without changing its meaning. If it can be, please break long lines here and in all other places